static/src/javascripts/projects/commercial/modules/dfp/define-slot.ts

```diff
@@ -135,12 +139,13 @@ const defineSlot = (
 	if (!slot) {
 		throw new Error(`Could not define slot for ${id}`);
 	}
+
 	/*
         For each ad slot defined, we request information from IAS, based
         on slot name, ad unit and sizes. We then add this targeting to the
         slot prior to requesting it from DFP.
 
-        We race the request to IAS with a Timeout of 2 seconds. If the
```

Comment:
Further down this function we set:

`const iasTimeoutDuration = 1000;`

I'm not a fan of these comments as they get out of date easily, but for now I have just updated it.

Reply:
Agreed, I'd be happy to drop the exact timeout  from the comment (now or later!)

Reply:
Sweet!